### PR TITLE
Add cws runtime anomaly detection docs

### PR DESCRIPTION
**Pull request by DominicBurkart**
### What does this PR do?

This PR adds documentation for Cloud Workload Security's new "Runtime Anomaly Detection" feature.

### Motivation

The feature is going to be GA'd for dash! 🚀 

https://docs-staging.datadoghq.com/dominic.burkart/cecile.guyon/cws-runtime-anomaly-detection/security/cloud_workload_security/runtime_anomaly_detection/

### Additional Notes

Our timeline for these docs are pretty tight: we need them for dash (and ideally sooner so we can have at least one iteration round). 

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/security/cloud_workload_security/runtime_anomaly_detection.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+## Overview
+
+*Available for containerized workloads running agent version 7.47 and above.*
+
+Runtime Anomaly Detection in Cloud Workload Security allows your security agents to observe processes spawned by your applications, along with network traffic and file accesses. It can then alert you when the runtime behavior of your containers change.
+
+By profiling your application as it runs, Runtime Anomaly Detection can detect attacks within containerized environments. Designed to integrate seamlessly with rule-based detections, Anomaly Detection helps aggregate suspicious activity directly from your agent, complementing rules that detect a wide array of attacks with detailed context on unusual behavior within your application’s environment. Anomaly Detection, unlike rule-based approaches, can identify when workloads:
+- launch new custom commands (such as cryptominers),
+- access local files for the first time, or
+- make new DNS resolutions (e.g. to exfiltrate data).
```
Comment (DominicBurkart) [inline content/en/security/cloud_workload_security/runtime_anomaly_detection.md:31]: NIT from contributing: https://github.com/DataDog/documentation/blob/master/CONTRIBUTING.md

```suggestion
- make new DNS resolutions (such as to exfiltrate data).
```

File: content/en/security/cloud_workload_security/runtime_anomaly_detection.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+- Suggested Next Steps provides essential pivots into Datadog observability tools such as Infrastructure Monitoring, access to the Security Profile to review known behavior for the service, and links to escalate the workflow using Case Management or Incidents, along with any custom-defined workflows. If the signal is benign, you can add it to the Security Profile of your workload to mark it as normal activity and avoid generating signals if the same behavior is observed again.
+- Anomaly Detected displays a list of Anomaly Events in chronological order. Here, we see that a shell script was downloaded, made executable, and launched.
+
+## Notifications
+
+Notifications can be used to alert you via slack or email when anomaly signals are generated. They can also be used to trigger webhooks or Datadog Workflows, enabling automated remediation pipelines. This is done with [Security Notification Rules][3]. To specifically target anomaly signals in a notification rule, use the rule criteria “rule_id:anomaly_detection”.
```
Comment (DominicBurkart) [inline content/en/security/cloud_workload_security/runtime_anomaly_detection.md:59]:
```suggestion
Notifications can be used to alert you via slack or email when anomaly signals are generated. They can also be used to trigger webhooks or Datadog Workflows, enabling automated remediation pipelines. This is done with [Security Notification Rules][3]. To specifically target anomaly signals in a notification rule, use the rule criteria `rule_id:anomaly_detection`.
```

File: content/en/security/cloud_workload_security/runtime_anomaly_detection.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+further_reading:
+- link: "/security/cloud_workload_security/setup"
+  tag: "Documentation"
+  text: "Setting Up CWS"
+- link: "/security/cloud_workload_security/security_profiles"
+  tag: "Documentation"
+  text: "CWS Security Profiles"
+---
+
+Runtime Anomaly Detection provides out of the box detection of unusual behavior on your _containerized_ workloads, using a model based on [CWS Security Profiles][6].
```
Comment (DominicBurkart) [inline content/en/security/cloud_workload_security/runtime_anomaly_detection.md:13]:
```suggestion
Runtime Anomaly Detection provides out of the box detection of unusual behavior on your containerized workloads, using a model based on [CWS Security Profiles][6].
```

File: content/en/security/cloud_workload_security/setup.md [hunk 4]
```diff
@@ -217,6 +238,16 @@ runtime_security_config:
     enabled: true
 ```
 
+**Optional, to enable Runtime Anomaly Detection** add the following field to the `system-probe.yaml` file:
```
Comment (drichards-87) [inline content/en/security/cloud_workload_security/setup.md:241]:
```suggestion
**Optional**: To enable Runtime Anomaly Detection, add the following to the `system-probe.yaml` file:
```

File: content/en/security/cloud_workload_security/setup.md [hunk 5]
```diff
@@ -106,6 +116,17 @@ Follow the [in-app instructions][6] in the Datadog app for the best experience,
           enabled: true
     ```
 
+    **Optional, to enable Runtime Anomaly Detection** add the following field to the `values.yaml` file:
```
Comment (drichards-87) [inline content/en/security/cloud_workload_security/setup.md:119]:
```suggestion
    **Optional** To enable Runtime Anomaly Detection, add the following to the `values.yaml` file:
```

File: content/en/security/cloud_workload_security/setup.md [hunk 6]
```diff
@@ -81,6 +81,16 @@ Follow the [in-app instructions][6] in the Datadog app for the best experience,
         runtime:
           enabled: true
     ```
+    **Optional, to enable Runtime Anomaly Detection** add the following field to the `values.yaml` file:
```
Comment (drichards-87) [inline content/en/security/cloud_workload_security/setup.md:84]:
```suggestion
    **Optional**: To enable Runtime Anomaly Detection, add the following to the `values.yaml` file:
```

File: content/en/security/cloud_workload_security/runtime_anomaly_detection.md [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+Notifications can be used to alert you via slack or email when anomaly signals are generated. They can also be used to trigger webhooks or Datadog Workflows, enabling automated remediation pipelines. This is done with [Security Notification Rules][3]. To specifically target anomaly signals in a notification rule, use the rule criteria `rule_id:anomaly_detection`.
+
+## Audit Trail for Anomaly Signals Actions
+
+As an administrator or security team member, you can use [Datadog Audit Trail][5] to see what actions your team is taking within the Cloud Security Product. As an individual, you can see a stream of your own actions. Audit trails for security signals are provided without additional configuration.
+
+
+### Navigate to Organization Settings and select Audit Trail under Compliance.
+
+To exclusively show Audit Logs generated by actions taken in Cloud Security Management:
```
Comment (drichards-87) [inline content/en/security/cloud_workload_security/runtime_anomaly_detection.md:70]:
```suggestion
To view audit logs generated by actions taken in Cloud Security Management, navigate to **Organization Settings** > **Compliance** > **Audit Trail** and do one of the following:
```

File: content/en/security/cloud_workload_security/runtime_anomaly_detection.md [hunk 8]
```diff
@@ -0,0 +1,86 @@
...
+- use the following query: `@evt.name:"Cloud Security Platform"`
+- or select the “Cloud Security Platform” facet under the “Event Name” facet.
```
Comment (drichards-87) [inline content/en/security/cloud_workload_security/runtime_anomaly_detection.md:72]:
```suggestion
- Create the following query: `@evt.name:"Cloud Security Platform"`
- In the **Event Name** facet, select **Cloud Security Platform**.
```

File: content/en/security/cloud_workload_security/runtime_anomaly_detection.md [hunk 9]
```diff
@@ -0,0 +1,86 @@
...
+
+To exclusively show Audit Logs generated by actions taken on security signals:
```
Comment (drichards-87) [inline content/en/security/cloud_workload_security/runtime_anomaly_detection.md:74]:
```suggestion
To view audit logs generated by actions taken on security signals:
```

File: content/en/security/cloud_workload_security/runtime_anomaly_detection.md [hunk 10]
```diff
@@ -0,0 +1,86 @@
...
+- use the following query: `@asset.type:security_signal`
+- or select the “security_signal” facet under the “Asset Type” facet.
```
Comment (drichards-87) [inline content/en/security/cloud_workload_security/runtime_anomaly_detection.md:76]:
```suggestion
- Create the following query: `@asset.type:security_signal`
- In the **Asset Type** facet, select **security_signal**..
```

File: content/en/security/cloud_workload_security/runtime_anomaly_detection.md [hunk 11]
```diff
@@ -0,0 +1,86 @@
...
+- Anomaly Detected displays a list of Anomaly Events in chronological order. Here, we see that a shell script was downloaded, made executable, and launched.
+
+## Notifications
+
+Notifications can be used to alert you via slack or email when anomaly signals are generated. They can also be used to trigger webhooks or Datadog Workflows, enabling automated remediation pipelines. This is done with [Security Notification Rules][3]. To specifically target anomaly signals in a notification rule, use the rule criteria `rule_id:anomaly_detection`.
+
+## Audit Trail for Anomaly Signals Actions
+
+As an administrator or security team member, you can use [Datadog Audit Trail][5] to see what actions your team is taking within the Cloud Security Product. As an individual, you can see a stream of your own actions. Audit trails for security signals are provided without additional configuration.
```
Comment (drichards-87) [inline content/en/security/cloud_workload_security/runtime_anomaly_detection.md:65]:
```suggestion
As an administrator or security team member, you can use [Datadog Audit Trail][5] to see what actions your team is taking in Cloud Security Management. As an individual, you can see a stream of your own actions. Audit trails for security signals are provided without additional configuration.
```

File: content/en/security/cloud_workload_security/runtime_anomaly_detection.md [hunk 12]
```diff
@@ -0,0 +1,86 @@
...
+
+
+### Navigate to Organization Settings and select Audit Trail under Compliance.
```
Comment (drichards-87) [inline content/en/security/cloud_workload_security/runtime_anomaly_detection.md:68]:
```suggestion
```

File: content/en/security/cloud_workload_security/runtime_anomaly_detection.md [hunk 13]
```diff
@@ -0,0 +1,86 @@
...
+- Suggested Next Steps provides essential pivots into Datadog observability tools such as Infrastructure Monitoring, access to the Security Profile to review known behavior for the service, and links to escalate the workflow using Case Management or Incidents, along with any custom-defined workflows. If the signal is benign, you can add it to the Security Profile of your workload to mark it as normal activity and avoid generating signals if the same behavior is observed again.
+- Anomaly Detected displays a list of Anomaly Events in chronological order. Here, we see that a shell script was downloaded, made executable, and launched.
+
+## Notifications
+
+Notifications can be used to alert you via slack or email when anomaly signals are generated. They can also be used to trigger webhooks or Datadog Workflows, enabling automated remediation pipelines. This is done with [Security Notification Rules][3]. To specifically target anomaly signals in a notification rule, use the rule criteria `rule_id:anomaly_detection`.
```
Comment (drichards-87) [inline content/en/security/cloud_workload_security/runtime_anomaly_detection.md:61]:
```suggestion
You can set up [security notification rules][3] to alert you via Slack or email when anomaly signals are generated. Notifications can also be used to trigger webhooks or Datadog Workflows, enabling automated remediation pipelines. To target anomaly signals in a notification rule, use the rule criteria `rule_id:anomaly_detection`.
```

File: content/en/security/cloud_workload_security/runtime_anomaly_detection.md [hunk 14]
```diff
@@ -0,0 +1,86 @@
...
+
+The Overview tab on Anomaly Detection Security Signals have three main sections, visible here:
+
+{{< img src="security/cws/security_profiles/runtime-anomaly-detection-signal-sidepanel.png" alt="A CWS runtime anomaly detection signal with three parts described below." width="50%">}}
+
+- What Happened explains which container was impacted and describes how to evaluate the signal. Here, we see that a container with the short image name “cws-webapp-v2” was impacted.
+- Suggested Next Steps provides essential pivots into Datadog observability tools such as Infrastructure Monitoring, access to the Security Profile to review known behavior for the service, and links to escalate the workflow using Case Management or Incidents, along with any custom-defined workflows. If the signal is benign, you can add it to the Security Profile of your workload to mark it as normal activity and avoid generating signals if the same behavior is observed again.
+- Anomaly Detected displays a list of Anomaly Events in chronological order. Here, we see that a shell script was downloaded, made executable, and launched.
+
+## Notifications
```
Comment (drichards-87) [inline content/en/security/cloud_workload_security/runtime_anomaly_detection.md:59]:
```suggestion
## Set up real-time notifications
```

File: content/en/security/cloud_workload_security/runtime_anomaly_detection.md [hunk 15]
```diff
@@ -0,0 +1,86 @@
...
+
+After a learning period, your agents can begin detecting anomalies. The learning duration varies depending on the workload’s activity, by default taking between 48 and 120 hours.
+
+As soon as the learning period is over, the agent begins evaluating new anomalous behavior and sending anomaly events to Datadog. Datadog’s platform aggregates these events by container and confirms that the anomalous behavior is not present in the Security Profile.
+
+If the behavior is not known in the profile, then the event (or events) are then surfaced as Security Signals. Signals are updated dynamically based on any newly detected anomalous behavior in the same container.
+
+
+
+## Inspect a Runtime Anomaly Signal
```
Comment (drichards-87) [inline content/en/security/cloud_workload_security/runtime_anomaly_detection.md:45]:
```suggestion
## Inspect runtime anomaly signals
```

File: content/en/security/cloud_workload_security/runtime_anomaly_detection.md [hunk 16]
```diff
@@ -0,0 +1,86 @@
...
+---
+
+Runtime Anomaly Detection provides out of the box detection of unusual behavior on your _containerized_ workloads, using a model based on [CWS Security Profiles][6].
+
+
+## Compatibility
+
+- Datadog Agent 7.47 or later.
+- [CWS enabled][1].
+- Runtime Anomaly Detection enabled: review [CWS setup instructions][2] to learn how to enable it.
```
Comment (drichards-87) [inline content/en/security/cloud_workload_security/runtime_anomaly_detection.md:20]:
```suggestion
```

File: content/en/security/cloud_workload_security/runtime_anomaly_detection.md [hunk 17]
```diff
@@ -0,0 +1,86 @@
...
+
+By profiling your application as it runs, Runtime Anomaly Detection can detect attacks within containerized environments. Designed to integrate seamlessly with rule-based detections, Anomaly Detection helps aggregate suspicious activity directly from your agent, complementing rules that detect a wide array of attacks with detailed context on unusual behavior within your application’s environment. Anomaly Detection, unlike rule-based approaches, can identify when workloads:
+- launch new custom commands (such as cryptominers),
+- access local files for the first time, or
+- make new DNS resolutions (e.g. to exfiltrate data).
+
+## How it works
+
+Once Runtime Anomaly Detection is activated, CWS agents start recording the normal behavior of your containerized workloads: commands run, file accesses, and DNS resolutions. This behavior is aggregated across all running containers with the same short image name and stored as Security Profiles.
```
Comment (drichards-87) [inline content/en/security/cloud_workload_security/runtime_anomaly_detection.md:35]:
```suggestion
Once you activate Runtime Anomaly Detection, the CWS Agents start recording the normal behavior of your containerized workloads. This includes commands run, files accessed, and DNS resolutions. This behavior is aggregated across all running containers with the same short image name and stored as Security Profiles.
```

File: content/en/security/cloud_workload_security/runtime_anomaly_detection.md [hunk 18]
```diff
@@ -0,0 +1,86 @@
...
+
+After a learning period, your agents can begin detecting anomalies. The learning duration varies depending on the workload’s activity, by default taking between 48 and 120 hours.
```
Comment (drichards-87) [inline content/en/security/cloud_workload_security/runtime_anomaly_detection.md:37]:
```suggestion
After a learning period, your Agents can begin detecting anomalies. The learning duration varies depending on the workload’s activity, by default taking between 48 and 120 hours.
```

File: content/en/security/cloud_workload_security/runtime_anomaly_detection.md [hunk 19]
```diff
@@ -0,0 +1,86 @@
...
+
+As soon as the learning period is over, the agent begins evaluating new anomalous behavior and sending anomaly events to Datadog. Datadog’s platform aggregates these events by container and confirms that the anomalous behavior is not present in the Security Profile.
```
Comment (drichards-87) [inline content/en/security/cloud_workload_security/runtime_anomaly_detection.md:39]:
```suggestion
As soon as the learning period is over, the Agent begins evaluating new anomalous behavior and sending anomaly events to Datadog. Datadog’s platform aggregates these events by container and confirms that the anomalous behavior is not present in the Security Profile.
```

File: content/en/security/cloud_workload_security/runtime_anomaly_detection.md [hunk 20]
```diff
@@ -0,0 +1,86 @@
...
+
+If the behavior is not known in the profile, then the event (or events) are then surfaced as Security Signals. Signals are updated dynamically based on any newly detected anomalous behavior in the same container.
```
Comment (drichards-87) [inline content/en/security/cloud_workload_security/runtime_anomaly_detection.md:41]:
```suggestion
If the behavior is not present in the security profile, the events are then surfaced as security signals. Signals are updated dynamically based on any newly detected anomalous behavior in the same container.
```

File: content/en/security/cloud_workload_security/runtime_anomaly_detection.md [hunk 21]
```diff
@@ -0,0 +1,86 @@
...
+
+
+
+## Inspect a Runtime Anomaly Signal
+
+*For more information about investigating Security Signals, see the Security [Signals Explorer docs][4].*
```
Comment (drichards-87) [inline content/en/security/cloud_workload_security/runtime_anomaly_detection.md:47]:
```suggestion
```

File: content/en/security/cloud_workload_security/runtime_anomaly_detection.md [hunk 22]
```diff
@@ -0,0 +1,86 @@
...
+
+The Overview tab on Anomaly Detection Security Signals have three main sections, visible here:
+
+{{< img src="security/cws/security_profiles/runtime-anomaly-detection-signal-sidepanel.png" alt="A CWS runtime anomaly detection signal with three parts described below." width="50%">}}
+
+- What Happened explains which container was impacted and describes how to evaluate the signal. Here, we see that a container with the short image name “cws-webapp-v2” was impacted.
+- Suggested Next Steps provides essential pivots into Datadog observability tools such as Infrastructure Monitoring, access to the Security Profile to review known behavior for the service, and links to escalate the workflow using Case Management or Incidents, along with any custom-defined workflows. If the signal is benign, you can add it to the Security Profile of your workload to mark it as normal activity and avoid generating signals if the same behavior is observed again.
```
Comment (drichards-87) [inline content/en/security/cloud_workload_security/runtime_anomaly_detection.md:56]:
```suggestion
The Overview tab on Anomaly Detection Security Signals have three main sections, visible here:

{{< img src="security/cws/security_profiles/runtime-anomaly-detection-signal-sidepanel.png" alt="A CWS runtime anomaly detection signal with three parts described below." width="50%">}}

- The **What Happened** section explains which container was impacted and describes how to evaluate the signal.
- **Suggested Next Steps** provides essential pivots into Datadog observability tools such as Infrastructure Monitoring, access to the Security Profile to review known behavior for the service, and links to escalate the workflow using Case Management or Incidents, along with any custom-defined workflows. If the signal is benign, you can add it to the Security Profile of the workload to mark it as normal activity and avoid generating signals if the same behavior is observed again.
```

**Review by drichards-87 — CHANGES_REQUESTED** [review]
Left you feedback from Docs. Do you want this article to appear in the left-nav Table of Contents? Currently, it isn't displayed there. If you'd like, I can pull the branch and add it to the left-nav.

File: content/en/security/cloud_workload_security/runtime_anomaly_detection.md (hunk 16 above)
[the remark this answers is not in the document]
Comment (DominicBurkart) [reply content/en/security/cloud_workload_security/runtime_anomaly_detection.md:20]: We need for them to add a config value. We can delete the "CWS enabled" section though!

**Comment by DominicBurkart** [comment]
@drichards-87 please do; this has already been approved by the CWS eng team and shared with our PMs, so feel free to merge this ASAP at your disgression.

**Review by drichards-87 — APPROVED** [review]

**Pull request merged**